README.md

```diff
@@ -4,165 +4,171 @@ The ATAT Web API is the internal application programming interface that supports
 ## ATAT Portfolio Drafts API
 The following steps describe the process for installing and deploying the ATAT Portfolio Drafts API.
 
-### Prerequisites:
-* Node Version Manager
-  ```
-  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
-  ```
-  Then restart your shell.\
-  Details available at [Installing and Updating](https://github.com/nvm-sh/nvm#installing-and-updating) in the nvm README.
-  
-* Node.js
-  ```
-  nvm install
-  ```
-  Node.js version is specified in file .nvmrc
-* Serverless Framework
-  ```
-  npm install -g serverless
-  ```
-* Typescript
-  ```
-  npm install -g typescript
-  ```
-* Azure CLI\
-  Follow steps at [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) at microsoft.com
+### Install Prerequisites
 ---
-### Deploying
-#### Install npm dependencies
-```
-cd ./poc/random_quote
-npm ci
-```
-
+These software packages are required.
+
+| Package | Command | Notes |
+| ----------- | ----------- |----------- |
+| Node Version Manager | `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh \| bash` | Then restart your shell. Details available in the [nvm README](https://github.com/nvm-sh/nvm#installing-and-updating). |
+| Node.js | `nvm install` | Node.js version is specified in file `./.nvmrc` |
+| Serverless Framework | `npm install -g serverless` | n/a |
+| TypeScript | `npm install -g typescript` | n/a |
+| Azure CLI | n/a | Follow steps at [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) at microsoft.com |
+| Terraform | n/a | Follow steps at [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/azure-get-started) at hashicorp.com |
+
+### Deploy Shared Infrastructure
 ---
+This monorepo currently contains a single Application (ATAT Portfolio Drafts API), but may eventually contain multiple Applications which all share some common infrastructure pieces. The following instructions describe their deployment:
+[Terraform](https://www.terraform.io/) will be used to deploy shared infrastructure in Microsoft Azure.
+Examples of such shared resources:
+* Key Vault
+* Storage Account
+* Cosmos DB Account and Database 
+* Azure Service Bus
 
-#### Login to Azure and set subscription
-```
-az login
-az account set -s <subscription-id>
-```
-
-
-## Shared Infrastructure
-Shared infrastructure for the PoC can be found at `./poc/shared_infrastructure`
-
-### Install dependencies
-For a detailed tutorial, follow the guidance here: [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/azure-get-started)
-
-### Local Deployment
-
-Navigate to the `./poc/shared_infrastructure` directory.
+The following is a list of `main.tf` files and their locations which can be used with Terraform to deploy Shared Infrastructure:
```

Comment:
We can tidy this up to just point to the single `main.tf` now

Reply:
Did this for navigation instructions in both areas where `./poc` references were removed.